### ISSUE TITLE
Existing baseline of ufs-weather-model cannot be reproduced when FMS2022.03 is used

### ISSUE THREAD

**Issue by binli2337** [ufs-community/ufs-weather-model#1386]
## Description
FMS2022.03 includes updates for mixed-mode applications. When FMS2022.03 is used in the develop branch of ufs-wearther-model, current baseline of regression tests cannot be reproduced. Before the release of FMS2022.03, FMS2022.03-alpha1 was installed on Gaea for testing and the baseline of ufs-weather-model could be completely reproduced. For details, see https://github.com/ufs-community/ufs-weather-model/issues/1036
### To Reproduce:
What compilers/machines are you seeing this with? Intel/Gaea

1. check out the most recent develop branch of ufs-weather-model
2. cd ufs-weather-model/modulefiles
3. modify ufs_common and ufs_common_debug by replacing fms/2022.01 with fms/2022.03
4. run any of the tests in rt.conf

**Comment by binli2337** [comment]
The following information is provide by Lauren Chilutti:
"In fms 2022.02 we introduced a new cmake flag -DCONSTANTS which by default is set to "GFDL". For the UFS weather model we need to specify -DCONSTANTS=GFS."

For testing, I installed FMS2022.03 library by adding the flag "-DCONSTANTS=GFS" to the existing cmake flags on Gaea. With this FMS2022.03 library, the current baseline of ufs-weather-model regression tests can be reproduced.
